### ISSUE TITLE
Replace try! in examples with ?

### ISSUE THREAD

**Issue by tirkarthi** [rust-lang/rust-www#631]
https://www.rust-lang.org/en-US/faq.html#how-do-i-read-a-file-into-a-string contains example with `try!` . The examples could make use of the new `?` operator. Similarly the following could be updated or a note could be added about the new operator : 

- https://www.rust-lang.org/en-US/faq.html#error-handling-without-result
- https://www.rust-lang.org/en-US/faq.html#why-do-i-get-errors-with-try

**Comment by AndrewBrinker** [comment]
It's an easy enough change to do. I'd like to be consistent with other Rust docs. Are there plans to move over Rust documentation in general to use `?` over `try!`?

**Comment by steveklabnik** [comment]
Yes, in general this should happen. We don't currently have a tracking
issue or anything.
